### DISCUSSION TITLE
計算機概論

### DISCUSSION THREAD

**Discussion by Clare0808** [1101-GS4719-C/Python-Community-GS4719-C#2027] [General]
https://m.gamer.com.tw/home/creationDetail.php?sn=3095871

之前沒有學過計算機概論，所以對於二進制、八進制這些並不熟悉，於是就上網稍微查了一下。

其中最簡單的是十進制，就是我們平常在用的進位法，而二進制基本上就是只有0和1在做變換。經過了0、1之後，十進制的2在二進制中會變成10，3會變成11。反正就是當所有的位數都被填滿1後，數字就必須進位，如111的下一個是1000。八進制就是把十進制的8、9直接跳過；十六進制則是從10開始要改成ABC。

其實文章的表格應該寫的很清楚了，稍微想一下就能理解。

以上是我的分享，提供給大家～

**Comment by Clare0808** [comment]
補充一下十六進制：0～9和十進制相同，10～15變成A~F。之後就是再從斷掉的地方（16）繼續數10個數字，然後第11個開始變成1A、1B……以此類推。

所以16變10，26變1A；32變20，42變2A。

**Comment by Piyenparty** [comment]
那我來分享一下，大一就有學到這種二進制代碼，但始終沒有實作過。在我這學期修習網概的時候，我發現ip位址的發放與分配，跟二進制編碼完全脫離不了關係( IPv4) ，自從腦海中的知識被實用，我就有一種踏實的感覺
